odo/backends/sql_csv.py

```diff
@@ -229,6 +230,101 @@ def append_csv_to_sql_table(tbl, csv, bind=None, **kwargs):
 
     kwargs = merge(csv.dialect, kwargs)
     stmt = CopyFromCSV(tbl, csv, bind=bind, **kwargs)
-    with bind.begin() as conn:
-        conn.execute(stmt)
+
+    if dialect == 'postgresql':
+        with bind.begin() as c:
+            with csv.open() as f:
+                c.connection.cursor().copy_expert(literal_compile(stmt), f)
+    else:
+        with bind.begin() as conn:
+            conn.execute(stmt)
     return tbl
+
+
+class NanIsNotNullFormatter(CSVFormatter):
```

Comment:
I see the test for this, where NaN's should stay NaN because spite of this wrapper- is there a test for the opposite of this? Where Null should be inserted correctly if not float?